README.md

```diff
@@ -7,7 +7,7 @@
 </p>
  :notes: Generating original Lofi-Music :notes:
 
-[COC](https://github.com/CommunityOfCoders/Inheritance-2023) Inheritance 2023</h3>
```

Comment:
Why this link was removed ?? This needs to be there I think. It's the place from where this amazing project originated.

Reply:
bro, I have removed it. He didn't pull the changes or something... that's why it's showing that he did that change.

Reply:
Oh. Okay. No problem. We'll add that later 😄 .